.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Test
+
+on:
+  pull_request:
+    branches: [main, development, feat/*]
+    types: [opened, ready_for_review]
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: Build
+        run: go build -v ./...
+
```

Comment:
i think we can remove this, since we already have the build check in the other workflow, or if it's for getting the dependencies, i think we can simply get them with `go get`